cmd/earthly/app/before.go

```diff
@@ -23,69 +23,74 @@ import (
 )
 
 func (app *EarthlyApp) before(cliCtx *cli.Context) error {
-	if app.BaseCLI.Flags().EnableProfiler {
+	flags := app.BaseCLI.Flags()
```

Comment:
nice change :)

Reply:
Thanks. It's a little easier on the eyes :slightly_smiling_face: